WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1200,8 +1200,12 @@ import WordPressComAnalytics
             }
 
             let failureBlock = { (error: Error?) in
+                guard let error = error else {
```

Comment:
The `as` is not conditional so it will always succeed. Doing that will mask `error: Error` with `error: NSError`. I think it's a bit clearer as it is now, but no big deal either way